### PR TITLE
added  home and docs page

### PR DESCRIPTION
**Pull request by GeetaPogul**

### CODE REVIEW

File: Docs/docs.html [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+       
+       
+                   <li  class="nav-li-links">   <span >  <button class="theme-button" id="theme-btn" > 
+                      <i class='fas fa-moon'> </i>   </button>
+                   </li>
+               </ul>
+         
+       </div>
+      </nav>
+
```
Comment (nshanbhag20001412) [inline Docs/docs.html:48]: you can remove the space here

File: home/home.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+
+.sub-content{
+    font-size:1.2rem;
+    /*padding: 1rem; */
+    margin: auto;
+    padding:  1rem 0 1.5rem ;
+    max-width:750px ;
+    width: 90%;
+    color: #333333;
```
Comment (nshanbhag20001412) [inline home/home.css]: you can add color variable here

Comment (nshanbhag20001412) [reply home/home.css]: also the main of your website should be named as the index.html. I assumed the home.html is your main page

**Review by sakshamAK — APPROVED** [review]

File: home/home.css (hunk 2 above)
Comment (GeetaPogul) [reply home/home.css]: Ok I do this.
Thank you for review.

**Pull request merged**